storage/sqlStorages/template.go

```diff
@@ -0,0 +1,175 @@
+package sqlStorages
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/config"
+	"github.com/cube2222/octosql/execution"
+	"github.com/cube2222/octosql/physical"
+	"github.com/cube2222/octosql/physical/metadata"
+	"github.com/pkg/errors"
```

Comment:
Reformat code, those should be split into 3 groups. Stdlib, external, and internal.